### DISCUSSION TITLE
Rimuovere vecchi sensori dopo rinominazione

### DISCUSSION THREAD

**Discussion by array81** [indomus/forum#1805] [HUB: Home Assistant]
Utilizzo un componente custom per monitorare il mio impianto fotovoltaico.
Per aggiungerlo a Home Assistant ho dovuto creare un sensore modificare il file configuration.yaml.
Ho deciso di rinominare il sensore modificando nuovamente configuration.yaml.
Ora in Home Assistant mi ritrovo sia il vecchio sensore che il nuovo. Il nuovo viene aggiornato mentre il vecchio no e mantiene la storia passata.
Come faccio a rimuovere il vecchio sensore? Nel file configuration.yaml non esiste più e se provo ad eliminarlo come entità mi viene detto che è sono in lettura. Il sensore inoltre "non ha un ID univoco, pertanto le sue impostazioni non possono essere gestite dall'interfaccia utente"

**Comment by 50494554524F** [comment]
toglie tutte le definizioni al vecchio sensore, riavvia, adesso dovrebbe comparire come non disponibile, può rimuoverlo...
